### PR TITLE
 fixing issue #31 with updated node docs #46 

### PR DESCRIPTION
**Pull request by SabineAuer**
To fix issue #31 I changed the docs for the FourthEq-model [1] by adding the explanation for the inertia-normalized frequency. Also I added the references for the inverter models for VSIMinimal [2] and VSIVoltagePT1 model [3]

    * [1] https://juliaenergy.github.io/PowerDynamics.jl/stable/node_types.html#PowerDynBase.FourthEq

    * [2] https://juliaenergy.github.io/PowerDynamics.jl/stable/node_types.html#PowerDynBase.VSIMinimal

    * [3] https://juliaenergy.github.io/PowerDynamics.jl/stable/node_types.html#PowerDynBase.VSIVoltagePT1

### CODE REVIEW

**Comment by timkittel** [comment]
1. Please build the docs and read through it before submitting. You'll see a few things that are not correct.
1. Also, as there are quite a lot of equations it might be good to give a short (1 sentence is enough) explantion of what each bunch of the equations actually does. The idea of the docs is that people can read it and quickly understand what is being done (assuming they have  certain background in power dynamics modeling).
1. Furthermore, there are quite a few unexplained dynamic variables.
1. We have conventions on how to call write parameters, please follow them.

File: src/NodeDynamics/FourthOrderEquation.jl [hunk 1]
```diff
@@ -6,25 +6,32 @@
...
+- `D`: damping coefficient (given in [s], see P. Sauer, eq. (5.156) where the damping torque is equal `Dω`)
 - `Ω`: rated frequency of the power grid, often 50Hz
-- `Ω_H`: normalized `Ω` according to ``Ω_H = (Ω * 2\pi) / H``
-- `T_d_dash`: time constant of d-axis
-- `T_q_dash`: time constant of q-axis
-- `X_d_dash`: transient reactance of d-axis
-- `X_q_dash`: transient reactance of q-axis
-- `X_d`: reactance of d-axis
-- `X_d`: reactance of q-axis
+- `Ω_H`: normalized `Ω` (given in [``s^{-2}``]) according to `Ω_H = (Ω⋅2\pi) / H`
```
Comment (timkittel) [inline src/NodeDynamics/FourthOrderEquation.jl]: This is not a Keyword argument, right? It's just a temporary variable computed in the code. It should be removed here. And I don't actually think there should be any further documentation of this variable except what is already directly in the code.

**Comment by timkittel** [comment]
Except for the one line that I just reviewed, I think that looks very good. Could you simply remove that?

Now, I would suggest we fix the ci problems with travis, then fix the conflicts with the (then) master branch and then merge. Do you agree, @SabineAuer ?

**Comment by SabineAuer** [comment]
That's fine for me. I'll remove the line and recommit to the branch. I agree that the fix of the travis CI should be high prio from now on.

**Comment by SabineAuer** [comment]
@timkittel: I merge the master into the branch of this PR so the travis CI can run through for this PR as well. Now all checks but the code coverage succeeded. The latter says:

>  Showing 1 files with coverage changes found. 

I am not sure what to do with this failed check. Can you help me out?

**Comment by timkittel** [comment]
Some how coverage seems to have gone down by one line. I am not sure, why that happens, but in this case I would say it's okai. Improving coverage is an issue anyway.

So from my point of view, you can force merge it.

**Comment by timkittel** [comment]
(Probably the total number of lines has gone down, which makes the absolute number of uncovered lines relatively higher.)

**Pull request merged**